sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveTableScanSuite.scala

```diff
@@ -187,6 +189,47 @@ class HiveTableScanSuite extends HiveComparisonTest with SQLTestUtils with TestH
     }
   }
 
+  test("SPARK-32867: When explain, HiveTableRelation show limited message") {
+    withSQLConf(HiveUtils.CONVERT_METASTORE_ORC.key -> "false",
```

Comment:
Is this config related to this test? Looks we can write tests without it though.

Reply:
> Is this config related to this test? Looks we can write tests without it though.

Just a  habit about writing UT of hive table. Remove it, thanks for correct this.